x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout_logic.ts

```diff
@@ -174,5 +193,9 @@ export const TextExpansionCalloutLogic = kea<
       (pollingTimeoutId: TextExpansionCalloutValues['textExpansionModelPollTimeoutId']) =>
         pollingTimeoutId !== null,
     ],
+    isStartButtonDisabled: [
+      () => [selectors.startTextExpansionModelStatus],
+      (status: Status) => status !== Status.IDLE,
```

Comment:
Are we sure we want to disable this button when we receive an error and block people from retrying?

Reply:
Error handling will be implemented once I rebase and adjust [this PR](https://github.com/elastic/kibana/pull/155774). I'll make sure the action is retryable upon an error.